src/is-authenticated-with-context.js

```diff
@@ -0,0 +1,10 @@
+import Cookies from 'js-cookie'
+
+export default function () {
```

Comment:
@davemcorwin No this definitely makes sense...I just made this more complicated than necessary. My thought was that perhaps in the future more than one "context type" might be needed. However, a single string would take care of the use case needed for InternX.